### ISSUE TITLE
browser logs with chrome.switches

### ISSUE THREAD

**Issue by hurjup** [serenity-bdd/serenity-core#1400]
Hello,

I've run into the same (unresolved?) problem as stated in https://github.com/serenity-bdd/serenity-core/issues/943.

OS: Windows10
Serenity version: 1.9.45

Properties file:
```
...
webdriver.driver=chrome
webdriver.chrome.driver=src/test/resources/webdrivers/chromedriver.exe
chrome.switches=--no-sandbox,--enable-logging=stdout,--v=4,--log-path=C:\\chromedriver1234.log
chrome_preferences.webdriver.chrome.logfile=C:\\chromedriver12345.log
...
```

However a process explorer claims

> C:\PF\cucumber\fx-automation\src\test\resources\webdrivers\chromedriver.exe --port=13261

being the actual command line of chrome driver, which correlates nicely with the serenity logs:

> Starting ChromeDriver 2.42.591088 (7b2b2dca23cca0862f674758c9a3933e685c27d5) on port 13261

Any idea what wend wrong?
Thank you in advance!

**Comment by wakaleo** [comment]
That is the correct way to provide chrome switches and chrome driver binary. Maybe put a breakpoint in ChromeServicePool to see how it is resolving the path.

**Comment by hurjup** [comment]
Meanwhile I managed bypassing the chrome log. I needed it to get the webSocketDebuggerUrl to establish a websocket connection to chrome DevTools, but I managed to assemble the URL using solely the information (port, handle) provided by the driver.get* methods. So I'm better curious now than being under pressure to get the logs.

My inspection regarding ChromeServicePool was less successful. All I managed to figure out is the presence of the port value, which confirms (sort of) the original problem statement.
But being more particular, I still don't really know how to catch when Serenity consumes values from the properties file.

**Comment by wakaleo** [comment]
Have a look in the `ChromeDriverCapabilities` class

**Comment by hurjup** [comment]
Thank you for the hint.
I've debugged the code and it was reassuring what I've seen, so for I've trusted in my Java skills. So, to doublecheck myself I've just wrote a short script that dumps the process tree being under my IDE (process name, pid, commandline). The "top -level chrome process" really has the commandline switch I've configured for logging... which was false according to https://www.chromium.org/for-testers/enable-logging .
This ticket is resolved (user error), so will be closed now.
